### PR TITLE
docs: Document usage of examples

### LINKED ISSUE #93: Add link to Jupyter notebook example in README
This will immediately show what can be done with our package.

### PR DESCRIPTION
**Pull request by mihaic**
Fixes #93, #228.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -101,6 +101,20 @@ as long as you see ``Successfully installed`` at the end.
 .. _issue #61:
    https://github.com/IntelPNI/brainiak/issues/61
 
+
+Examples
+========
+
+We include BrainIAK usage examples in the examples directory of the code
```
Comment (yidawang) [inline README.rst]: We should point out that this is an example of SRM in BrainIAK. For other components currently we also have example codes available but not in Notebook format.

Comment (mihaic) [inline README.rst]: I think the name shown there is self explanatory: "srm_image_prediction_example". And we do have notebooks for other examples as well.

Comment (yidawang) [inline README.rst]: then how about providing more example links?

Comment (mihaic) [inline README.rst]: I would rather keep the README short. This is only an invitation for users to look in the examples directory.

Comment (yidawang) [inline README.rst]: OK, sounds good

**Pull request merged**